docs/api/web-contents.md

```diff
@@ -987,12 +987,12 @@ Returns `String` - The user agent for this web page.
 
 **[Deprecated](modernization/property-updates.md)**
 
-#### `contents.insertCSS(css)`
+#### `contents.insertCSS(css[, origin])`
 
 * `css` String
+* `origin` String (optional) - Can be either 'user' or 'author'; Specifying "user" enables you to prevent websites from overriding the CSS you insert. Default is 'author'.
```

Comment:
Can we make this an object in case we want to add more options here like the chrome API